src/pages/nfts/[address].tsx

```diff
@@ -38,9 +41,17 @@ import { seededRandomBetween } from '@/modules/utils/random';
 import { SolscanIcon } from '@/assets/icons/Solscan';
 import { ExplorerIcon } from '@/assets/icons/Explorer';
 import NFTFile from '@/components/NFTFile';
+import { ClipboardCheckIcon, ExclamationCircleIcon } from '@heroicons/react/outline';
 import { ButtonSkeleton } from '@/components/Skeletons';
-// import { Tag as FeatherTag, Zap } from 'react-feather';
+import { DollarSign, Tag as FeatherTag, Zap } from 'react-feather';
+import Popover from '../../components/Popover';
 import { LightningBoltIcon, TagIcon } from '@heroicons/react/outline';
+import { getAuctionHouseInfo } from '../../modules/utils/marketplace';
+
+// TODO: update sdk to include marketplaceProgramAddress
+export interface AhListingMultiMarketplace extends AhListing {
+  marketplaceProgramAddress?: string;
+}
```

Comment:
It only takes a sec to publish the marketplace-js-sdk. I recommend you update that first then use here.